### DISCUSSION TITLE
Question: Is AKHQ for Kafka 3.0 version available?

### DISCUSSION THREAD

**Discussion by Art-master** [tchiotludo/akhq#1128] [Q&A] [answered]
Hi @tchiotludo !
Kafka 3.0 is available, it does not use Zookeeper. Can I use AKHQ with this version of Kafka?
As far as I know, AKHQ rely on ZooKeeper and I cannot install it without ZooKeeper. 
Or do you plan to upgrade to support new versions of Kafka?

**Comment by tchiotludo** [comment, marked as answer]
Hi @Art-master, 

Response is simple : 
- AKHQ is compatible with all version of Kafka has I know even 3.0+ 
- AKHQ never rely on zookeeper, only the docker-compose have still a zookeeper because kraft isn't GA for now

So feel to free to enjoy AKHQ with Kafka 3.0+

**Reply by Art-master** [reply]
Thanks for the answer, tchiotludo
